packages/babel-traverse/src/path/family.ts

```diff
@@ -193,11 +193,16 @@ function getStatementListCompletion(
     }
   } else if (paths.length) {
     // When we are in a context where `break` must not exist, we can skip linear
-    // search on statement lists and assume that the last statement determines
-    // the completion
-    completions = completions.concat(
-      _getCompletionRecords(paths[paths.length - 1], context),
-    );
+    // search on statement lists and assume that the last
+    // non-variable-declaration statement determines the completion.
+    for (let i = paths.length - 1; i >= 0; i--) {
+      if (!paths[i].isVariableDeclaration()) {
+        completions = completions.concat(
+          _getCompletionRecords(paths[paths.length - 1], context),
```

Comment:
```suggestion
          _getCompletionRecords(paths[i], context),
```
Guess it should have been `paths[i]`.